packages/sources/elwood/src/endpoint/crypto.ts

```diff
@@ -18,27 +14,14 @@ const inputParameters = new InputParameters(priceEndpointInputParametersDefiniti
   },
 ])
 
-type OmitResultFromLwba = Omit<LwbaResponseDataFields, 'Result'>
-
 export type BaseEndpointTypes = {
   Parameters: typeof inputParameters.definition
   Settings: typeof config.settings
-  Response: OmitResultFromLwba & SingleNumberResultResponse
+  Response: SingleNumberResultResponse
```

Comment:
This is not backward compatible so should be a major version bump?

Reply:
Good point. I don't think it's necessary to actually make this change so I'll revert the response to it's original format